config/initializers/honeycomb.rb

```diff
@@ -0,0 +1,39 @@
+require 'honeycomb-beeline'
+
+module CCInitializers
```

Comment:
what's up with the module here? does this run at process startup or something?

Reply:
Yeah, this is our version of a standard Rails pattern. The NewRelic integration has a slightly more involved version where it only runs the initializers at all if NewRelic is enabled.